src/content/contributing/design/index.md

```diff
@@ -58,7 +58,7 @@ Our design system makes designing ethereum.org fun and easy. If you are an exper
 
 ### <Emoji text=":six:" size={1} /> Write design-related content on the website {#write-design-articles}
 
-The Ethereum developer community is strong, but the design community is falling slightly behind. If you are a designer with web3 knowledge, please consider sharing your learnings with the larger community so that we can all grow and improve together!
+The Ethereum developer community is strong, but the design community is falling slightly behind. If you are a designer with web3 knowledge, please consider sharing your learnings with the larger community so that we can all grow and improve together! We have a [design section](/developers/docs/design-and-ux/) that needs more articles!
```

Comment:
```suggestion
The Ethereum developer community is strong, but the design community is falling slightly behind. If you are a designer with web3 knowledge, please consider sharing your learnings with the larger community so that we can all grow and improve together, we have a [design section](/developers/docs/design-and-ux/) that needs more articles!
```